README.dk.md

```diff
@@ -0,0 +1,66 @@
+# Sway
+
+Sway er en [i3](https://i3wm.org/)-kompatibel [Wayland](http://wayland.freedesktop.org/) typograf.
+Læs [Ofte stillede spørgsmål](https://github.com/swaywm/sway/wiki).
+Deltag på [IRCkanalen](http://webchat.freenode.net/?channels=sway&uio=d4) (#sway på irc.freenode.net).
+
+Hvis du har lyst til at støtte sway udviklingen, venlist send bidrag på [SirCmpwn'sPatreon side](https://patreon.com/sircmpwn).
```

Comment:
Is it intentional to not use spaces in the [SirCmpwn'sPatreon side] link text?

Reply:
non intentional, thanks for noticing.
will --amend with a g in venligst.